launch.sh

```diff
@@ -196,8 +200,15 @@ fail_if_root
 
 print_delimiter
 
-# Get the absolute path of the directory where this script is located
-SRC_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# Check if any arguments were supplied and launch wizard if not
+WIZARD_FILE_DIR="${SCRIPT_DIR}/wizard.sh"
+if [[ $# -eq 0 ]]; then
+    if [[ ! -f $WIZARD_FILE_DIR ]]; then
```

Comment:
How about we move this to L229 and replace that if block with this logic? Same idea where if there are no arguments provided, we launch the wizard.

Reply:
definitely, it is also good because that way all of the requirements are installed and configured before the wizard initializes, will implement now

Reply:
also, we might need to keep the edge case since we don't have a default value for ORIGIN_DIR ill push the idea now

Reply:
Ok i implemented it please review again @harryarakkal